DEVGUIDE.md

```diff
@@ -116,10 +116,10 @@ Note: if you face this error [#2351](https://github.com/Microsoft/visualfsharp/i
 
 >  error VSSDK1077: Unable to locate the extensions directory. "ExternalSettingsManager::GetScopePaths failed to initialize PkgDefManager for C:\Program Files (x86)\Microsoft Visual Studio\2017\Community\Common7\IDE\devenv.exe".
 
-delete those folders:
+Or hard crash on launch ("Unknown Error"), delete these folders:
 
-* %appdata%\Microsoft\VisualStudio\15.0_<some number here>FSharpDev
-* %appdata%\..\Local\Microsoft\VisualStudio\15.0_<some number here>FSharpDev
+* `%appdata%\Microsoft\VisualStudio\15.0_(some number here)FSharpDev`
+* `%localappdata%\..\Local\Microsoft\VisualStudio\15.0_(some number here)FSharpDev`
```

Comment:
Sorry I wasn't very clear I think this should be: 
```
%localappdata%\Microsoft\VisualStudio\15.0_(some number here)FSharpDev
%localappdata%\Microsoft\VisualStudio\15.0_(some number here)FSharpDev
```
Folder looks like:
![image](https://cloud.githubusercontent.com/assets/5175830/24076285/10dfcfc0-0bea-11e7-9988-0991a1321c0a.png)